src/qt/guiutil.cpp

```diff
@@ -129,15 +129,26 @@ void setupAddressWidget(QValidatedLineEdit *widget, QWidget *parent) {
     widget->setCheckValidator(new BitcoinAddressCheckValidator(parent));
 }
 
+static bool IsCashAddrEncoded(const QUrl &uri) {
+    const std::string addr = (uri.scheme() + ":" + uri.path()).toStdString();
+    auto decoded = cashaddr::Decode(addr, "");
+    return !decoded.first.empty();
+}
+
 bool parseBitcoinURI(const QString &scheme, const QUrl &uri,
                      SendCoinsRecipient *out) {
     // return if URI has wrong scheme.
-    if (!uri.isValid() || uri.scheme() != scheme) {
+    if (!uri.isValid() || !(uri.scheme() == scheme || uri.scheme() == "payto" || uri.scheme() == "")) {
```

Comment:
Why does the "payto" here have to be hard-coded?

Reply:
What should it be?

Reply:
I was thinking a constant somewhere

Reply:
I was thinking a constant somewhere

Reply:
I added a constant in the header file.